core/vibes/soul/primitives/favorite/index.tsx

```diff
@@ -22,9 +23,10 @@ export interface FavoriteProps {
  * }
  * ```
  */
-export const Favorite = ({ checked = false, setChecked }: FavoriteProps) => {
+export const Favorite = ({ checked = false, setChecked, label = 'Favorite' }: FavoriteProps) => {
   return (
     <Toggle.Root
+      aria-label={label}
```

Comment:
The [first rule of `aria`](https://www.w3.org/TR/using-aria/#firstrule) is to use built-in behavior before we use `aria-` attributes. We should be able to utilize the svg's `<title>` tag here instead:
```tsx
<Heart filled={checked}>
  <title>{label}</title>
</Heart>
```